apps/shipments/views.py

```diff
@@ -252,3 +254,26 @@ def list(self, request, *args, **kwargs):
             return paginator.get_paginated_response(page)
 
         return Response(serializer.data)
+
+
+class ListDevicesStatus(APIView):
+    http_method_names = ['get', ]
+    permission_classes = (permissions.IsAuthenticated, )
+    pagination_class = CustomResponsePagination
+    renderer_classes = (renderers.JSONRenderer,)
+
+    def get(self, request, *args, **kwargs):
+        owner_id = get_owner_id(request)
+        iot_client = DeviceAWSIoTClient()
+
+        serializer = DevicesQueryParamsSerializer(data={'in_bbox': request.query_params.get('in_bbox'),
```

Comment:
Since the parameters are already named the same, can't you just do `data=request.query_params`?